README.md

```diff
@@ -60,30 +60,33 @@ class motd {
     mode  => '0644'
   }
 
-  concat::fragment{ 'motd_header':
+  concat::fragment { 'motd_header':
     target  => $motd,
     content => "\nPuppet modules on this server:\n\n",
     order   => '01'
   }
 
   # let local users add to the motd by creating a file called
   # /etc/motd.local
-  concat::fragment{ 'motd_local':
+  concat::fragment { 'motd_local':
     target => $motd,
     source => '/etc/motd.local',
     order  => '15'
   }
 }
 
 # let other modules register themselves in the motd
-define motd::register($content="", $order='10') {
+define motd::register (
+  $content="",
```

Comment:
There should be spaces around the `=` for all of these parameters as well:
```
define motd::register (
  $content = '',
  $order   = '10',
) {
```

Reply:
@natemccurdy Thank you for the hint. I've updated the PR.